corehq/apps/hqadmin/views.py

```diff
@@ -458,6 +459,49 @@ def page_context(self):
         return context
 
 
+class VCMMigrationView(BaseAdminSectionView):
+    urlname = 'vcm_migration'
+    page_title = ugettext_lazy("Vellum Case Management Migration")
```

Comment:
Yeah, I see your point, hadn't thought about that. I think I still favor referencing "vellum" over calling it "Case Management in the Form Builder Migration" for the sake of space.